socorro/external/fs/crashstorage.py

```diff
@@ -176,16 +171,18 @@ def _save_files(self, crash_id, files):
                 # probably already created, ignore
                 pass
 
-            for fn, contents in files.iteritems():
+            for fn, contents in files.items():
```

Comment:
I ditched all the `iteritems` for `items`. In all of these cases, the dicts are really small, so using `items` isn't a big deal.